pkg/model/resources/nodeup.go

```diff
@@ -66,8 +66,13 @@ download-or-bust() {
     for url in "${urls[@]}"; do
       local file="${url##*/}"
       rm -f "${file}"
-      if ! curl -f --ipv4 -Lo "${file}" --connect-timeout 20 --retry 6 --retry-delay 10 "${url}"; then
-        echo "== Failed to download ${url}. Retrying. =="
+
+      if [[ $(which curl) ]]; then
```

Comment:
just a note.  We need to allow for s3cmd and gs :)

Reply:
I think both can do signed URLs.